pinc/Project.inc

```diff
@@ -554,6 +554,32 @@ class Project
         return $this->can_be_managed_by_current_user || $this->PPer_is_current_user;
     }
 
+    function get_num_pages_in_state($state=NULL, $counter="*")
```

Comment:
might want to explicitly call out that this is sql and won't be encoded in a query by naming it something like $counter_sql or something perhaps?

Reply:
Good call - done.